core/components/molecules/verificationCodeInput/VerificationCodeInput.tsx

```diff
@@ -0,0 +1,264 @@
+    if (value.length > 1) {
+      let nextIndex = value.length + index - 1;
+      if (nextIndex >= fields) {
+        nextIndex = fields - 1;
+      }
+      next = iRefs[nextIndex];
+      const split = value.split('');
+      split.forEach((item: string, i: number) => {
+        const cursor: number = index + i;
+        if (cursor < fields) {
+          valuesCopy[cursor] = item as never;
+        }
+      });
+      setValues(valuesCopy);
+    } else {
+      next = iRefs[index + 1];
+      valuesCopy[index] = value as never;
+      setValues(valuesCopy);
+    }
+
+    if (next && next.current) {
+      next.current.focus();
+      next.current.select();
+    }
+
+    triggerChange(e, valuesCopy);
+  };
+
+  const onFocusHandler = (e: FocusEvent<HTMLInputElement>) => {
+    e.target.select();
+    e.target.placeholder = '';
+    if (onFocus) {
+      onFocus(e);
+    }
+  };
+
+  const onBlurHandler = (e: FocusEvent<HTMLInputElement>) => {
+    e.target.placeholder = placeholder;
+    if (onBlur) {
+      onBlur(e);
+    }
+  };
+
+  const onKeyDown = (e: KeyboardEvent<HTMLInputElement>) => {
+    const index = parseInt(e.currentTarget.dataset.id as string, 10);
+    const prevIndex = index - 1;
+    const nextIndex = index + 1;
+    const prev: RefObject<HTMLInputElement> = iRefs[prevIndex];
+    const next: RefObject<HTMLInputElement> = iRefs[nextIndex];
+    switch (e.key) {
+      case KEY_CODE.backspace:
+        e.preventDefault();
+        const vals = [...values];
+        if (values[index]) {
+          vals[index] = '' as never;
+          setValues(vals);
+          triggerChange(e as unknown as ChangeEvent<HTMLInputElement>, vals);
+        } else if (prev && prev.current) {
+          vals[prevIndex] = '' as never;
+          prev.current.focus();
+          setValues(vals);
+          triggerChange(e as unknown as ChangeEvent<HTMLInputElement>, vals);
+        }
+        break;
+      case KEY_CODE.left:
+        e.preventDefault();
+        if (prev && prev.current) {
+          prev.current.focus();
+        }
+        break;
+      case KEY_CODE.right:
+        e.preventDefault();
+        if (next && next.current) {
+          next.current.focus();
+        }
+        break;
+      case KEY_CODE.up:
+      case KEY_CODE.down:
+        e.preventDefault();
+        break;
+      case KEY_CODE.e:
+      case KEY_CODE.E:
+        if (type === 'number') {
+          e.preventDefault();
+        }
+        break;
+      default:
+        break;
+    }
+  };
+
+  const getId = (i: number) => (id ? `${id}-${i}` : undefined);
+  const wrapperClassNames = (i: number) =>
+    classNames({
+      'VerificationCodeInput-wrapper': true,
+      'ml-3': i > 0
+    });
+
+  return (
+    <div data-test="DesignSystem-VerificationCodeInput">
+      {values.map((value: string, index: number) => (
+        <div
+          key={`${id}-${index}`}
+          className={wrapperClassNames(index)}
+        >
```

Comment:
Please remove `div` wrapper and add className 'VerificationCodeInput-input' to the Input component only.